src/csvcubed/models/cube/qb/components/codelist.py

```diff
@@ -91,6 +112,14 @@ def __post_init__(self):
             self.concept_scheme_uri = None  # type: ignore
             self.concept_template_uri = None  # type: ignore
 
+    def _get_validations(self) -> Dict[str, ValidationFunction]:
+        return {
+            "schema_metadata_file_path": validate_file,
+            "csv_file_relative_path_or_uri": validate_uri,
+            "concept_scheme_uri": validate_uri,
+            "concept_template_uri": validate_uri,
```

Comment:
I'm not 100% certain that validate_uri will is the right validation to apply against concept_template_uri. I think the location in which its used (gss-utils) will provide relative paths/URIs which won't look like a complete URI.

Reply:
Changed to validate_str